src/org/usfirst/frc/team2706/robot/subsystems/DriveTrain.java

```diff
@@ -29,37 +30,39 @@
  * These include four drive motors, a left and right encoder and a gyro.
  */
 public class DriveTrain extends Subsystem {
-    private WPI_TalonSRX front_left_motor, back_left_motor, front_right_motor, back_right_motor;
+    private EWPI_TalonSRX front_left_motor, back_left_motor, front_right_motor, back_right_motor;
     private DifferentialDrive drive;
     private TalonEncoder left_encoder, right_encoder;
-    private Ultrasonic leftDistanceSensor, rightDistanceSensor;
     private AHRS gyro;
 
     // TODO: maybe we don't need this
     private GyroPIDSource gyroPIDSource;
     private AverageEncoderPIDSource encoderPIDSource;
-    private UltrasonicPIDSource ultrasonicPIDSource;
     private TalonPID talonPID;
 
     public double initGyro;
 
     private Command defaultCommand;
 
-    // The spinny dial on the robot that selects what autonomous mode we are going to do
-    private AutonomousSelector selectorSwitch;
-
     public DriveTrain() {
         super();
-        front_left_motor = new WPI_TalonSRX(RobotMap.MOTOR_FRONT_LEFT);
-        back_left_motor = new WPI_TalonSRX(RobotMap.MOTOR_REAR_LEFT);
-        front_right_motor = new WPI_TalonSRX(RobotMap.MOTOR_FRONT_RIGHT);
-        back_right_motor = new WPI_TalonSRX(RobotMap.MOTOR_REAR_RIGHT);
+        front_left_motor = new EWPI_TalonSRX(RobotMap.MOTOR_FRONT_LEFT);
+        back_left_motor = new EWPI_TalonSRX(RobotMap.MOTOR_REAR_LEFT);
+        front_right_motor = new EWPI_TalonSRX(RobotMap.MOTOR_FRONT_RIGHT);
+        back_right_motor = new EWPI_TalonSRX(RobotMap.MOTOR_REAR_RIGHT);
 
         front_left_motor.setInverted(RobotMap.MOTOR_FRONT_LEFT_INVERTED);
         back_left_motor.setInverted(RobotMap.MOTOR_REAR_LEFT_INVERTED);
         front_right_motor.setInverted(RobotMap.MOTOR_FRONT_RIGHT_INVERTED);
         back_right_motor.setInverted(RobotMap.MOTOR_REAR_RIGHT_INVERTED);
-
+        
+        front_left_motor.configPeakCurrentLimit(2, 0);
```

Comment:
This doesn't actually work, so it should be looked at again.

Reply:
Yes, but it is a good idea if we will use different kinds of drive mechanisms....